chainer/functions/connection/n_step_lstm.py

```diff
@@ -454,6 +465,13 @@ def n_step_lstm(
 
     """
 
+    argument.check_unexpected_kwargs(
+        kwargs, train='train argument is not supported anymore. '
+        'Use chainer.using_config',
+        use_cudnn='use_cudnn argument is not supported anymore. '
+        'Use chainer.using_config')
+    argument.parse_kwargs(kwargs)
```

Comment:
OK. For this purpose, i'll make another method which only checks if kwargs is empty.